packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -23,6 +23,12 @@
 
 namespace DataTransferKit
 {
+namespace Details
+{
```

Comment:
I'm not sure I understand. This namespace only applies to a single declaration of `TreeTraversal`. The code for it lives in the `details` subdir, as expected.